yaptide/batch/batch_methods.py

```diff
@@ -327,3 +327,27 @@ def delete_job(simulation: BatchSimulationModel, user: KeycloakUserModel,
         return {"message": "Job cancelation failed"}, 500
 
     return {"message": "Job canceled"}, 200
+
+
+def fetch_results_without_stopping(simulation: BatchSimulationModel, user: KeycloakUserModel,
+                                   cluster: ClusterModel) -> tuple[dict, int]:  # skipcq: PYL-W0613
+    """Function that stops simulation without cancelling collection job"""
```

Comment:
Maybe in the name you should say something like `cancel_simulation_with_result_dump` saying in docstring that you only cancel simulation (array job) without cancelling merge job which should process all dumped files